fdbserver/storageserver.actor.cpp

```diff
@@ -683,6 +684,15 @@ struct StorageServer {
 						(currentRate() < 1e-6 ? 1e6 : 1.0 / currentRate()));
 	}
 
+	Future<Void> getQueryDelay() {
```

Comment:
Maybe add a comment like "
Lower query priority if the SS's durability version lag is too large or its write queue size is too large.
"

It helps future readers who are not reviewer of this PR understand why this function is added.